src/Components/signUp/SignUp.js

```diff
@@ -0,0 +1,236 @@
+import { Fragment, useState, useEffect } from "react";
+import { Navigate } from "react-router-dom";
+
+import { Button, TextField, InputAdornment } from "@mui/material";
+import VisibilityIcon from "@mui/icons-material/Visibility";
+import { VisibilityOff } from "@mui/icons-material";
+
+import Image from "../image/ImageContainer";
+import { styles } from "../../utils";
+import "./styleSignUp.scss";
+const textFieldBorderStyle = {
```

Comment:
lets create a file called `materialStyles.js` and in that file we are gonna move this object and export it, then import it within this file

Reply:
Ok, thank you.